templates/commands/render_action.go

```diff
@@ -120,10 +120,18 @@ func templateAndCompileRegexes(regexes []model.String, inputs map[string]string)
 
 // templateFuncs returns a function map for adding functions to go templates.
 func templateFuncs() template.FuncMap {
-	return map[string]interface{}{
-		"contains":          strings.Contains,
-		"replace":           strings.Replace,
-		"replaceAll":        strings.ReplaceAll,
+	return map[string]any{
+		"contains":   strings.Contains,
+		"replace":    strings.Replace,
+		"replaceAll": strings.ReplaceAll,
+		"sortStrings": func(in []string) []string {
+			// The default sort.Strings behavior modifies the string in place. I
+			// thought it would be very weird if rendering a template changed the
+			// order of an input further down the stack.
+			cp := append([]string{}, in...)
```

Comment:
Maybe slices.Copy() here to make this easier to read?

Reply:
It's `slices.Clone`, but sure. Done.